crates/biome_service/src/settings.rs

```diff
@@ -724,12 +724,11 @@ impl VcsSettings {
 
     /// Returns whether the given `path` should be ignored per the VCS settings.
     #[inline]
-    pub fn is_ignored(&self, path: &Utf8Path) -> bool {
+    pub fn is_ignored(&self, root_path: &Utf8Path, path: &Utf8Path) -> bool {
         self.should_use_ignore_file()
-            && self
-                .ignore_matches
-                .as_ref()
-                .is_some_and(|ignored_matches| ignored_matches.is_ignored(path, is_dir(path)))
+            && self.ignore_matches.as_ref().is_some_and(|ignored_matches| {
+                ignored_matches.is_ignored(path, root_path, is_dir(path))
```

Comment:
It looks like the arguments are reversed here too. `is_ignored()` takes the root path as first argument, and the path to query second. It could be argued though the signatures are actually confusing :)